src/openrct2/ride/Vehicle.h

```diff
@@ -150,6 +150,28 @@ enum VEHICLE_STATUS
 
 struct rct_vehicle_sound_params;
 
+enum VEHICLE_TRACK_SUBPOSITION : uint8_t
+{
+    VEHICLE_TRACK_SUBPOSITION_0,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_1,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_2,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_END_3,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_END_4,
+    VEHICLE_TRACK_SUBPOSITION_GO_KART_5,
+    VEHICLE_TRACK_SUBPOSITION_GO_KART_6,
+    VEHICLE_TRACK_SUBPOSITION_GO_KART_7,
+    VEHICLE_TRACK_SUBPOSITION_GO_KART_8,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_START_9 = 9,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_PATH_A_9 = 9,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_BALL_PATH_A_10,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_PATH_B_11,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_BALL_PATH_B_12,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_PATH_C_13,
+    VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_BALL_PATH_C_14,
+    VEHICLE_TRACK_SUBPOSITION_REVERSER_RC_15,
```

Comment:
15 is Rear bogie, 16 is front bogie I can't quite confirm which way round they are though as it doesnt make 100% sense.